Flutter 7 GA (#6468)
getsentry/sentry-docs: src/platforms/flutter/common/performance/instrumentation/automatic-instrumentation.mdx

### http.Client Library Instrumentation

> The `http.Client` instrumentation, once added the `SentryHttpClient` and enabled the [performance](/platforms/flutter/performance/) feature, starts a span out of the active transaction bound to the scope for each HTTP Request. The SDK sets the span `operation` to `http.client` and `description` to request `$method and $URL`; for example, `GET https://sentry.io`.

Reviewer: ```suggestion
Once enabled, the `http.Client` instrumentation adds the `SentryHttpClient` and enables the [performance](/platforms/flutter/performance/) feature. This starts an active transaction span, which is bound to the scope for each HTTP Request. The SDK then sets the span `operation` to `http.client` and the `description` to request `$method and $URL`; for example, `GET https://sentry.io`
```
Same comment as above, I'm not sure I'm fully understanding this correctly. Happy to talk about it more.

To start getting the Mobile Vitals data, you have to set up the [routing instrumentation](/platforms/flutter/performance/instrumentation/automatic-instrumentation/#routing-instrumentation) or the [GoRouter instrumentation](/platforms/flutter/performance/instrumentation/automatic-instrumentation/#gorouter-instrumentation).

> If you want to opt out, you can flip the `enableAutoPerformanceTracing` flag:

Reviewer: ```suggestion
To opt out, can flip the `enableAutoPerformanceTracing` flag:
```
